### ISSUE TITLE
No descriptions for resource links on resources page

### ISSUE THREAD

**Issue by jchristgit** [python-discord/site#1392]
As outlined in #1363:

The button to go to a specific resource at the bottom left of each card lacks any description and just reads as "link" with screen readers. This means that people using screen readers will not know where links will take them. They would have to activate the link to find out, whereas sighted users could visually identify Github and Goodreads icons, for example.

Links should describe where they will take the user. The “aria-label” attribute can be defined for each link to describe what it’s for. Alternatively, because the title of each card is a link to the same place, consider making the element hidden from screen readers entirely with “aria-hidden”, or moving the icon to the same element as the title.
